packages/core/types/src/tax/common.ts

```diff
@@ -364,31 +364,6 @@ export interface TaxableItemDTO {
    */
   product_id: string
 
-  /**
-   * The name of the item's product.
-   */
-  product_name?: string
-
-  /**
-   * The ID of the category of the item's product.
-   */
-  product_category_id?: string
-
-  /**
-   * The categories of the item's product.
-   */
-  product_categories?: string[]
-
-  /**
-   * The SKU of the item's product.
-   */
-  product_sku?: string
-
-  /**
-   * The type of the item's product.
-   */
-  product_type?: string
-
```

Comment:
These are currently not used for tax calculations so I removed them from the type and input.